env.json

```diff
@@ -0,0 +1,16 @@
+{
+  "sentryTransportLogger": {
+    "sentry": {
+      "dsn": "https://examplePublicKey@o0.ingest.sentry.io/0"
+    },
+    "level": "warn",
+    "levelsMap": {
+      "silly": "debug",
+      "verbose": "debug",
+      "info": "info",
+      "debug": "debug",
+      "warn": "warning",
+      "error": "error"
+    }
+  }
+}
```

Comment:
I think this file shouldn't be added in the repo, we should add something like `env.sample.json` and recommend users to rename it to `env.json`, and `env.json` should be added to gitignore. Can we call this `sentry-env.json`?